Apps/Common/test/unit/Services/PersonalAccountServiceTests.cs

```diff
@@ -82,31 +79,6 @@ public void ShouldGetPatientAccountUseCache()
             Assert.Equal(id, actualResult.ResourcePayload?.Id);
         }
 
-        /// <summary>
```

Comment:
Where is the unit test to handle ApiException and HttpRequestException.  Won't his have on code coverage?